pkg/asset/manifests/cloudproviderconfig.go

```diff
@@ -92,7 +93,7 @@ func (cpc *CloudProviderConfig) Generate(dependencies asset.Parents) error {
 	}
 
 	switch installConfig.Config.Platform.Name() {
-	case libvirttypes.Name, nonetypes.Name, baremetaltypes.Name, ovirttypes.Name:
+	case libvirttypes.Name, nonetypes.Name, baremetaltypes.Name, ovirttypes.Name, nutanixtypes.Name:
```

Comment:
I am a bit surprised that there is not a cloud provider config. Is the name of the secret with the creds hard-coded somewhere?

Reply:
Currently, yes. `nutanix-credentials` is the credential name.